config/application.rb

```diff
@@ -190,6 +190,14 @@ class Application < Rails::Application
     config.logger = Foreman::Logging.logger('app')
     config.active_record.logger = Foreman::Logging.logger('sql')
 
+    if config.serve_static_assets
```

Comment:
It may seem like that, by as far as I can tell, if I set it to false, the assets are not loaded. If I set it to true, the assets are loaded. Maybe what you are seeing is it being parsed but not executed?